templates/workspace_services/health-services/terraform/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "healthcare_workspace_id" {
```

Comment:
Can the FHIR and DICOM URIs be provided as outputs so can be copied via the portal?

Reply:
added